2_Operations/next-task.md

```diff
@@ -1,19 +1,18 @@
 # Find Your Next Task
```

Comment:
This should now be just going down the Backlog, right?

Reply:
I would simplify this whole section and just say to look at the Backlog and/or ask the product owner.

Reply:
I've simplified it to the section below and added it to the `kanban.md` file.
```makdown
## Find Your Next Task

Make sure you are completing your assigned tasks first since they are our current focus. If you've finished all of them, go through the issues in our [backlog] and or ask the project lead what to do next.

If you're a developer, you can also search the codebase and documentation for "TODO", "XXX" and similar strings. Try to understand why it's there and create an issue for it. If you like the task, you can start working on it immediately.

If all else is completed, you can always read a book, take an online course or just think about how we could make the company better.
```